hbase-server/src/main/jamon/org/apache/hadoop/hbase/tmpl/regionserver/RegionListTmpl.jamon

```diff
@@ -207,6 +211,21 @@
                                               * TraditionalBinaryPrefix.KILO.value, "B", 1);
               }
            }
+           Long lenOfBiggestCellInRegion = -1l;
```

Comment:
Why use Long instead of long here? And please use 1L instead of 1l, upper case...

Reply:
Will fix it, thanks a lot.